.github/workflows/daily.yml

```diff
@@ -34,17 +34,17 @@ jobs:
         run: |
           # needed for tabmat
           echo "Install compilation dependencies"
-          micromamba install -y c-compiler cxx-compiler cython jemalloc-local libgomp mako xsimd
+          micromamba install -y c-compiler cxx-compiler 'cython!=3.0.4' jemalloc-local libgomp mako xsimd
 
-          PRE_WHEELS="https://pypi.anaconda.org/scipy-wheels-nightly/simple"
+          PRE_WHEELS="https://pypi.anaconda.org/scientific-python-nightly-wheels/simple/"
```

Comment:
Adding a review comment so that we do not merge this right now. We need to fix the errors caused by the newer versions.